contracts/contracts/othellofactory.sol

```diff
@@ -174,17 +177,22 @@ contract othellofactory is Board{
             }
         }
         removeUsersFromGame();
-        emit EndGame(blackaddress,whiteaddress);
         
-        if(white==black){
-            return(blackaddress, true);
-        }
-        winner=black>white?blackaddress:whiteaddress;
+        _isDraw=white==black?true:false;
```

Comment:
_isDraw=white==black?true:false;

This ternary expression is unnecessary. The result of `white==black` is already a boolean, so you could write:

    _isDraw = white == black;

Reply:
yeah, you are right. Dumb me!